src/openstudio_app/CMakeLists.txt

```diff
@@ -156,6 +156,13 @@ if(WIN32)
   )
 endif()
 
+if(UNIX AND NOT APPLE)
+  add_custom_command(TARGET ${target_name}
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E make_directory $<TARGET_FILE_DIR:${target_name}>/xcbglintegrations/
+  )
+endif()
```

Comment:
Make `/path/to/build/Products/xcbglintegrations` so that we can copy the .so into it without getting a failure later.